corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -115,6 +115,7 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
             var menuData = menusUtils.getMenuData(menuResponse);
             menuData["triggerEmptyCaseList"] = true;
             menuData["sidebarEnabled"] = true;
+            menuData["title"] = menuResponse.resultsTitle;
```

Comment:
this is the use case where a default empty case list is rendered with the resultsTitle rather than the query response title used